extensions/podman/src/extension.ts

```diff
@@ -517,13 +542,21 @@ async function monitorProvider(provider: extensionApi.Provider) {
       if (!installedPodman) {
         provider.updateStatus('not-installed');
         extensionApi.context.setValue('podmanIsNotInstalled', true, 'onboarding');
+        if (isLinux() && shouldNotifySetup) {
```

Comment:
imho i think we should have comments here regarding why we are not notifying on linux. it's a bit ambiguous on here.